packages/@okta/vuepress-site/docs/guides/authenticators-okta-email/main/index.md

```diff
@@ -168,17 +172,33 @@ The Email Magic Links feature is designed with security in mind and only works w
 
 </div>
 
-<StackSnippet snippet="integratediffbrowserdevicesummary"/> <!-- UNIQUE FOR EACH LANGUAGE -->
+The following step-by-step instructions detail integrating the different browser scenario using the email challenge.
```

Comment:
is step-by-step really needed here? I feel like this is just extra wording that doesn't provide a lot of value or help.
```suggestion
The following instructions detail integrating the different browser scenario using the email challenge.
```

Reply:
updated.